src/pages/workspaces/workspace/WorkspaceAcl.ts

```diff
@@ -0,0 +1,41 @@
+import _ from 'lodash/fp';
+import { workspaceAccessLevels } from 'src/libs/utils';
```

Comment:
Having these in 2 different files is super confusing. Can we consolidate in a single location?

Reply:
Missed this comment earlier

Mind if I do that in the next PR? My local branch has a bunch of changes, and I don't want to have to work through the headache of linting partial commits or resolving conflicts when re-applying the stash.

Reply:
I think you actually took care of this now.